### PR TITLE
URW 191 Componentize member cards

### PR DESCRIPTION
**Pull request by Kenneth-W-Chen**
This PR is to componentize the `/about/our-team`, `/about/our-dev-team`, and `/about/our-alumni` pages in order to make it easier to add/remove members from these pages. Before, you had to dig through the HTML to find specific info about the member. Now, this info is quickly identifiable in an array at the top.

The CSS for `.bio-area` has been moved to `style.css` since it's used in these 3 pages.

This PR also removes the anchors from the `/about/our-dev-team` page because it wasn't consistent with the other pages (`our-alumni` did not have any, and `our-team` had empty anchors).

Also, I removed the template comment at the bottom of the dev team page, since it won't be needed to add new devs to the site.

Besides the above changes, none of the page contents or designs have been changed.

# tl;dr
* Page layout/design is the same
* Repeated HTML is now outputted through PHP and foreach loops
* Moved repeated CSS to our stylesheet

### CODE REVIEW

File: about/our-alumni.php [hunk 1]
```diff
@@ -1,14 +1,79 @@
...
+];
+$members[] = [
+    'name'=>'Jesse Sullivan',
+    'title'=>'Former Aerospace Division Lead',
+    'description'=>"Jesse is a senior mechanical engineering student interested in all things that fly and go fast. He has 4 years of experience in amateur high-power rocketry and currently holds an L1 certification with the National Association of Rocketry. He founded the Aerospace Division with UNT Robotics in order to foster interest in the hobby and provide a learning experience for anyone to become a part of.",
+    'picture_uri'=>'/images/bio-pics/jesse-sullivan.jpg',
+//    'email'=>'',
+//    'linkedin_url'=>'',
+//    'github_url'=>'',
+//    'twitter_url'=>''
```
Comment (sebastian-king) [inline about/our-alumni.php:74]: All these comments are just a lot of dead code, we should remove these

File: template/functions/card.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?php
```
Comment (sebastian-king) [inline template/functions/card.php:1]: Anything under the `/template` directory should only be globally applicable utilities, i.e. something that we could PR back into the framework repo. This file should be moved under the `/about` directory

File: css/style.css [hunk 3]
```diff
@@ -48054,4 +48054,11 @@ header nav.rd-navbar-static.rd-navbar-secondary .rd-navbar-nav > li.thin {
...
-/*# sourceMappingURL=style.css.map */
\ No newline at end of file
+/*# sourceMappingURL=style.css.map */
+
+/*Adds a line above the bios in about/our-team, about/our-dev-team, and about/our-alumni*/
+.bio-area {
+  border-top: 2px solid #ececec;
+  margin-top: 10px;
+  padding-top: 10px;
+}
```
Comment (sebastian-king) [inline css/style.css:48064]: The source mapping line needs to stay at the bottom of the file

**Review by sebastian-king — APPROVED** [review]

**Review by lmb-boy — APPROVED** [review]

**Review by tjcrozier — APPROVED** [review]
